docs/product-installation-guide/src/main/asciidoc/clustering-custom-config-proc.adoc

```diff
@@ -0,0 +1,67 @@
+[id='cluster-custom-config-proc']
+
+= Clustering {PRODUCT} on {EAP}
+
+When using {EAP} clustering, a single {EAP} domain controller exists with other {EAP} slaves connecting to it as management users. You can deploy {CENTRAL} as a management user on a domain controller, and the WAR deployments will be distributed to other members of the {EAP} cluster.
+
+.Prerequisite
+* A {EAP} cluster is available. 
+
+.Procedure
+
+. Define the data source driver.
++
+.PostgreSQL Driver Definition
+====
+[source,xml]
+----
+<driver name="postgres" module="org.postgresql">
+  <xa-datasource-class>org.postgresql.xa.PGXADataSource</xa-datasource-class>
+</driver>
+----
+====
++
+Ensure that the data sources contain schemas. If your data source does not contain schemas, ensure your nodes start one at a time.
+
+.
+// change on purpose - cd = change directory
+. Change to `_EAP_HOME_/bin` and start the application server in domain mode:
++
+On UNIX systems:
++
+----
+./domain.sh
+----
++
+On Windows:
++
+----
+./domain.bat
+----
+
+. Check that the nodes are available.
+
+Deploy the {CENTRAL} application to your servers:
+
+ifdef::PAM[]
+. Change the predefined persistence of the application to the required database (PostgreSQL): in `persistence.xml` change the following:
```

Comment:
there is no need to modify persistence.xml file any more in v7 - actually it is embedded in a jar file so it won't be easy to actually modify it. Instead following system properties should be set to direct to data source and hibernate dialect:

```
org.kie.server.persistence.ds
org.kie.server.persistence.dialect
```